packages/workspace/package.json

```diff
@@ -42,7 +42,9 @@
     "chalk": "^4.1.0",
     "enquirer": "~2.3.6",
     "tslib": "^2.3.0",
-    "yargs-parser": "21.1.1"
+    "yargs-parser": "21.1.1",
+    "picomatch": "4.0.2",
```

Comment:
Please can we put these in the correct order to avoid an unnecessary diff later?

Reply:
We should probably add a rule to enforce this programmatically

Reply:
Sure. I added them by using the `lint --fix`, so we should also look into having the rule add packages in the right order.